parent/pom.xml

```diff
@@ -144,6 +144,42 @@
     <version.jna-platform>5.14.0</version.jna-platform>
     <version.msal4j>1.15.0</version.msal4j>
 
+    <version.commons-io>2.16.0</version.commons-io>
```

Comment:
it seems that the properties in this file were sorted at some point, it can help finding duplicate lib usage